packages/fxa-payments-server/src/components/PlanDetails/index.stories.tsx

```diff
@@ -47,99 +53,93 @@ const selectedPlan = {
   },
 };
 
-storiesOf('components/PlanDetail', module)
-  .add('default', () => (
-    <MockApp>
-      <PlanDetails
-        {...{
-          profile: userProfile,
-          showExpandButton: false,
-          selectedPlan,
-          isMobile: false,
-        }}
-      />
-    </MockApp>
-  ))
-  .add('localized to xx-pirate', () => (
-    <MockApp languages={['xx-pirate']}>
-      <PlanDetails
-        {...{
-          profile: userProfile,
-          showExpandButton: false,
-          selectedPlan,
-          isMobile: false,
-        }}
-      />
-    </MockApp>
-  ))
-  .add('with expand button', () => (
-    <MockApp>
-      <PlanDetails
-        {...{
-          profile: userProfile,
-          showExpandButton: true,
-          selectedPlan,
-          isMobile: false,
-        }}
-      />
-    </MockApp>
-  ))
-  .add('with coupon - type "forever"', () => (
-    <MockApp>
-      <PlanDetails
-        {...{
-          profile: userProfile,
-          showExpandButton: false,
-          selectedPlan,
-          isMobile: false,
-          coupon: { ...COUPON_DETAILS_VALID, type: 'forever' },
-        }}
-      />
-    </MockApp>
-  ))
-  .add('with coupon - type "once"', () => (
-    <MockApp>
+type PlanDetailsProps = {
+  selectedPlan: Plan;
+  isMobile: boolean;
+  showExpandButton?: boolean;
+  className?: string;
+  coupon?: CouponDetails;
+  children?: any;
+};
```

Comment:
Q: Would it be better/easier to export the `PlanDetailsProps` type from index.tsx and then import it, versus duplicating it in the Storybook file?
https://github.com/mozilla/fxa/blob/d84af5eee500ca68d1f39dbf150216826046d9be/packages/fxa-payments-server/src/components/PlanDetails/index.tsx#L25-L32

Reply:
Hahahaha, this is the thought I just had and comment I made in Meghan's PR. Yes, I'll update this.